broker/aws/impl.go

```diff
@@ -705,7 +705,7 @@ func (b *Broker) generateTokenCredentials(accountName string, roleName string, u
 		Region:       region,
 		Expiration:   time.Now().Add(time.Second * profileAssumeRoleDurationSeconds),
 	}
-	b.auditLogger.Printf("Token credentials generated for: %s on account %s role %s", userName, accountName, roleName)
+	b.auditLogger.Printf("Token credentials generated for: %s on account %s role %s keyIdd %s", userName, accountName, roleName, outVal.SessionId)
```

Comment:
Perhaps log this as `sessionId`, to match the field?

Reply:
did minor cleanup (sessionId is actually a misnomer coming from the original aws broker).

Reply:
addressed in latest PR.